requirements.txt

```diff
@@ -1 +1,3 @@
-
+beautifulsoup4==4.10.0
+pytest==7.1.0
```

Comment:
you do not need pytest to run scrapper 
all the required dependencies for testing are located in requirements_qa.txt

Reply:
Removed